dll/win32/browseui/addresseditbox.cpp

```diff
@@ -318,8 +318,16 @@ HRESULT STDMETHODCALLTYPE CAddressEditBox::Execute(long paramC)
     hr = psf->CompareIDs(0, pidl, pidlLastParsed);
 
     SHFree(pidl);
-    if (hr == 0)
+
+    if (hr == S_OK)
```

Comment:
Comparing against `0` is cleaner here imo.
Apart from that, I don't know why there was code storing the last parsed pidl, and how that is influenced by this change.
@JoachimHenze can you elaborate on that?

Reply:
```suggestion
    if (hr == 0)
```

Reply:
No I can't elaborate on that. The code is from @KrosUser. Not from me. I gave my approval based on the promised functionality, and because I could not spot formal issues in the suggested patch.

Reply:
> Comparing against 0 is cleaner here imo.

What's the reasoning behind this?

Reply:
Because it's a comparison result.

Reply:
ah ok, makes sense then